databases/mongo/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mongo:bionic
+
+COPY fixtures /docker-entrypoint-initdb.d/
+
+ENV MONGO_INITDB_ROOT_USERNAME forest
+ENV MONGO_INITDB_ROOT_PASSWORD secret
+ENV MONGO_INITDB_DATABASE forest
```

Comment:
We should not name the database forest as it will be really confusing with our database which is totally different ^^